esp-hal/MIGRATING-0.22.md

```diff
@@ -143,4 +172,11 @@ is enabled. To retrieve the address and size of the initialized external memory,
 +let (start, size) = esp_hal::psram::psram_raw_parts(&peripherals.PSRAM, psram);
 ```
 
-The usage of `esp_alloc::psram_allocator!` remains unchanged.
+## Interrupt related reshuffle
+
+```diff
+- use esp_hal::InterruptConfigurable;
```

Comment:
Can we land this separately?

Reply:
Of course, I'll pull this out of this PR.